contentcuration/contentcuration/utils/celery_signals.py

```diff
@@ -66,6 +89,7 @@ def on_failure(sender, **kwargs):
 
 @task_success.connect
 def on_success(sender, result, **kwargs):
+    clear_and_connect()
```

Comment:
Same question here -- if we're always clearing connections before they run, do we still need to clear them here? Might just be extra work.

Reply:
The reason I added them after is that the errors we were seeing in sentry were frequently in the on_success triggers, and not in the task itself.

The `clear_and_connect` does not always reset the connection, it first checks that you can open a cursor to the DB, and then only resets the connection if that results in an InterfaceError.

Reply:
Ah I see! Sounds good on `on_success`.

Reply:
Can you add a single line comment then on why we need to do a `clear_and_connect` here?